instrumentation/net/http/splunkhttp/config.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkhttp
+
+import (
+	"os"
+	"strings"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+)
+
+// Environmental variables used for configuration.
+const (
+	EnvVarServerTimingEnabled = "SPLUNK_CONTEXT_SERVER_TIMING_ENABLED" // Adds `Server-Timing` header to HTTP responses
+)
+
+// WithOTelOpts is use to pass the OpenTelemetry SDK options.
+func WithOTelOpts(opts ...otelhttp.Option) Option {
+	return optionFunc(func(cfg *config) {
+		cfg.OTelOpts = opts
+	})
+}
+
+// WithServerTiming enabled or disabled the ServerTimingMiddleware.
```

Comment:
```suggestion
// WithServerTiming enables or disables the ServerTimingMiddleware.
//
// The default is to enable the ServerTimingMiddleware if this option is not passed.
// Additionally, the SPLUNK_CONTEXT_SERVER_TIMING_ENABLED environment
// variable can be set to TRUE or FALSE to specify this option. This option value will
// be given precedence if both it and the environment variable are set.
```

Reply:
Fixed https://github.com/signalfx/splunk-otel-go/pull/30/commits/165c57c14f22bd11afdca17c60347dcc4b9c7208

Reply:
Did you not want to add the paragraph about defaults and overrides with the environment variable?

Reply:
You mean https://github.com/signalfx/splunk-otel-go/blob/165c57c14f22bd11afdca17c60347dcc4b9c7208/instrumentation/net/http/splunkhttp/README.md ?

Reply:
Hmm, not sure I follow.

I had originally suggested the `s/(en|dis)abled/\1ables/` change which looks a part of [165c57c](https://github.com/signalfx/splunk-otel-go/commit/165c57c14f22bd11afdca17c60347dcc4b9c7208), but also suggested we add the following as well to the `WithServerTiming` documentation:

```
// The default is to enable the ServerTimingMiddleware if this option is not passed.
// Additionally, the SPLUNK_CONTEXT_SERVER_TIMING_ENABLED environment
// variable can be set to TRUE or FALSE to specify this option. This option value will
// be given precedence if both it and the environment variable are set.
```

Reply:
It is me how was not following. Give me a minute, please and sorry for inconvenience  😉

Reply:
Added here: https://github.com/signalfx/splunk-otel-go/pull/30/commits/092496b8b3a7ff804173dc420f8fb27e1b145cb9